charts/prometheus-node-exporter/Chart.yaml

```diff
@@ -6,8 +6,8 @@ keywords:
   - prometheus
   - exporter
 type: application
-version: 4.5.2
-appVersion: 1.3.1
```

Comment:
to use the app version you would need to adjust the deamonset too.
see: https://github.com/prometheus-community/helm-charts/blob/d3bccc205efd0a4dfa7fccc4cda25cf28272c6c6/charts/prometheus-node-exporter/templates/daemonset.yaml#L40

Reply:
Thank you, I think I looked at the wrong chart when I changed the `appVersion` 😅

Reply:
It seems to be already defined here https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-node-exporter/templates/_helpers.tpl#L75-L83